proxy.config.json

```diff
@@ -1,5 +1,16 @@
 {
-  "proxyUrl": "https://github.com",
+  "proxyConfig": [
+    {
+      "path": "/github.com",
+      "url": "https://github.com",
+      "enabled": true
+    },
+    {
+      "path": "/gitlab.com",
+      "url": "https://gitlab.com",
+      "enabled": true
```

Comment:
I like your proposed configuration. Would it not be better to recommend `/github` and `/gitlab`? I realise that this is configurable so doesn't really matter and gives control and choice to the developer.